picard/metadata.py

```diff
@@ -30,6 +30,21 @@
 
 MULTI_VALUED_JOINER = '; '
 
+# lengths difference over this number of milliseconds will give a score of 0.0
+# equal lengths will give a score of 1.0
+# example
+# a     b     score
+# 20000 0     0.333333333333
+# 20000 10000 0.666666666667
+# 20000 20000 1.0
+# 20000 30000 0.666666666667
+# 20000 40000 0.333333333333
+# 20000 50000 0.0
+LENGTH_SCORE_THRES_MS = 30000
+
+def length_score(a, b):
```

Comment:
Not sure if this should be a public functions, maybe _length_score or maybe move this to picard.util?

Reply:
I'll make it a static method